airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestinationResolver.java

```diff
@@ -37,6 +38,16 @@ public static boolean isAzureBlobCopy(final JsonNode config) {
         && config.get("loading_method").has("azure_blob_storage_account_name");
   }
 
+  public static int getNumberOfFileBuffers(final JsonNode config) {
+    int numOfFileBuffers = FileBuffer.DEFAULT_MAX_CONCURRENT_STREAM_IN_BUFFER;
+    if (config.has(FileBuffer.FILE_BUFFER_COUNT_KEY)) {
+      numOfFileBuffers = Math.min(config.get(FileBuffer.FILE_BUFFER_COUNT_KEY).asInt(), FileBuffer.MAX_CONCURRENT_STREAM_IN_BUFFER);
+    }
```

Comment:
Non-blocking but maybe a ticket should be filed to add this warning as well here

https://github.com/airbytehq/airbyte/blob/0b156a8326f5566c14063e424951f58c9495187a/airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftStagingS3Destination.java#L146-L152

It's to safeguard users from running into OOM issues by logging the the increased memory size that comes with more FileBuffers